sdk/python/jobs/pipelines/1h_automl_in_pipeline/automl-forecasting-in-pipeline/automl-forecasting-in-pipeline.ipynb

```diff
@@ -121,57 +372,62 @@
     "):\n",
     "    # define forecasting settings\n",
     "    forecasting_settings = ForecastingSettings(\n",
-    "        time_column_name=\"timeStamp\",\n",
-    "        forecast_horizon=12,\n",
-    "        frequency=\"H\",\n",
-    "        target_lags=[12],\n",
-    "        target_rolling_window_size=4,\n",
+    "        time_column_name=time_column_name,\n",
+    "        forecast_horizon=n_test_periods,\n",
+    "        frequency=\"W-THU\",\n",
+    "        cv_step_size=1,\n",
     "    )\n",
     "\n",
     "    # define the automl forecasting task with automl function\n",
     "    forecasting_node = forecasting(\n",
     "        training_data=forecasting_train_data,\n",
-    "        target_column_name=\"demand\",\n",
+    "        target_column_name=target_column_name,\n",
     "        primary_metric=\"normalized_root_mean_squared_error\",\n",
     "        n_cross_validations=2,\n",
     "        forecasting_settings=forecasting_settings,\n",
     "        # currently need to specify outputs \"mlflow_model\" explictly to reference it in following nodes\n",
```

Comment:
Change the comment to use CUSTOM_MODEL instead of mlflow_model.

Reply:
Fixed.